src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -39,68 +39,86 @@ public override void OnCreate()
 
 			var applicationContext = rootContext.MakeApplicationScope(this);
 
-			Services = applicationContext.Services;
+			_services = applicationContext.Services;
 
-			Current.Services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationCreating>(del => del(this));
+			_services.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationCreating>(del => del(this));
 
-			Application = Services.GetRequiredService<IApplication>();
+			_application = _services.GetRequiredService<IApplication>();
 
-			this.SetApplicationHandler(Application, applicationContext);
+			this.SetApplicationHandler(_application, applicationContext);
 
-			Current.Services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationCreate>(del => del(this));
+			_services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationCreate>(del => del(this));
 
 			base.OnCreate();
 		}
 
 		public override void OnLowMemory()
 		{
-			Current.Services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationLowMemory>(del => del(this));
+			_services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationLowMemory>(del => del(this));
 
 			base.OnLowMemory();
 		}
 
 		public override void OnTrimMemory(TrimMemory level)
 		{
-			Current.Services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationTrimMemory>(del => del(this, level));
+			_services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationTrimMemory>(del => del(this, level));
 
 			base.OnTrimMemory(level);
 		}
 
 		public override void OnConfigurationChanged(Configuration newConfig)
 		{
-			Current.Services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationConfigurationChanged>(del => del(this, newConfig));
+			_services?.InvokeLifecycleEvents<AndroidLifecycle.OnApplicationConfigurationChanged>(del => del(this, newConfig));
 
 			base.OnConfigurationChanged(newConfig);
 		}
 
 		public static MauiApplication Current { get; private set; } = null!;
 
-		public IServiceProvider Services { get; protected set; } = null!;
+		IServiceProvider _services = null!;
```

Comment:
Maybe this should be nullable. It should never be actually, except maybe if things are not set up correctly yet. So like pre-maui-app-starting it will be null... like in an embedded scenario. The field that is. 

We probably should also maybe throw an exception if it is null in the getters, but that is too risky just to throw in. Who knows what things we are checking for nulls. The annotation is probably not 100% but better than a crash due to a new exception.

Reply:
Same with all the platfoms. Might as well make the field right.

Reply:
done